cicd/gitlab/dot.gitlab-ci.yml

```diff
@@ -0,0 +1,28 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+image: python:3.8
+
+before_script:
+  - python --version  # For debugging
+  - python -m venv venv
+  - source venv/bin/activate
+
+test:
+  script:
+    - pip install mypy
+    - mypy src/python
```

Comment:
Do you mind adding `pylint` also to the first version of our CI/CD? Cheers!

Reply:
it's just a test. a failed one. I'll drop it